Libraries/Image/Image.ios.js

```diff
@@ -108,19 +108,34 @@ const ImageViewManager = NativeModules.ImageViewManager;
  * You will need to add some optional modules in `android/app/build.gradle`, depending on the needs of your app.
  *
  * ```
+ * // use getFrescoComponent to lock your components to the version used by react-native
```

Comment:
derp not `getFrescoComponent` but `getFrescoVersion` I can update when we're sure we want to move forward with this.